eleventy.config.js

```diff
@@ -39,7 +39,7 @@ module.exports = function(eleventyConfig) {
   // Usage: {% breadcrumbs page.url %}
   eleventyConfig.addShortcode("breadcrumbs", function(url) {
     const parts = url.split("/").filter((part) => part.length)
-    const links = ["<a href='/'>Home</a>"]
+    const links = ["<a href='/sprint-guide'>Home</a>"]
```

Comment:
The breadcrumbs was broken for me. It seems the redirect set here is not working:

https://github.com/thoughtbot/design-sprint-guide/blob/main/netlify.toml#L7

Or is that config only set for production?

Reply:
I don't know enough about this to be able to answer this question or this [one](https://github.com/thoughtbot/design-sprint-guide/pull/113/files#r1571518389). Other than that, I reviewed your changes and didn't see anything that stood out to me. LGTM 👍

Reply:
Thank you, Danny! I will merge this as it is, then. These links were not working for me locally.